advanced_html_css/animation/transforms.md

```diff
@@ -150,7 +150,7 @@ While you can generally chain multiple transforms in any order for various resul
 
 ### Three-dimensional transforms
 
-The `rotate`, `scale`, and `translate` transform functions aren't limited to just the 2D plane. They work for the 3D plane as well! However, to perceive a 3D effect on some of these function values, `perspective` is required.
+The `rotate`, `scale`, and `translate` transform functions aren't limited to just the 2D plane. They work for the 3D space as well! However, to perceive a 3D effect on some of these function values, `perspective` is required.
```

Comment:
The rest of the lesson uses the wording 3D plane, so I think it would be confusing to use the word space here. What do you think about changing it to something like this:
```suggestion
The `rotate`, `scale`, and `translate` transform functions aren't limited to just 2D planes. They also work for 3D planes as well! However, to perceive a 3D effect on some of these function values, `perspective` is required.
```

Reply:
A plane typically refers to a flat 2D object, whereas when discussing 3D transformations, it's more accurate to use the term '3D space.' I've gone ahead and replaced all instances of '3D plane' with '3D space' in the file. I hope this adjustment is appropriate and will improve the clarity of the content.